pandasai/constants.py

```diff
@@ -2,74 +2,19 @@
 Constants used in the pandasai package.
 """
 
-START_CODE_TAG = "<startCode>"
-END_CODE_TAG = "<endCode>"
-WHITELISTED_LIBRARIES = [
-    "numpy",
-    "matplotlib",
-]
-WHITELISTED_BUILTINS = [
-    "abs",
-    "all",
-    "any",
-    "ascii",
-    "bin",
-    "bool",
-    "bytearray",
-    "bytes",
-    "callable",
-    "chr",
-    "classmethod",
-    "complex",
-    "delattr",
-    "dict",
-    "dir",
-    "divmod",
-    "enumerate",
-    "filter",
-    "float",
-    "format",
-    "frozenset",
-    "getattr",
-    "hasattr",
-    "hash",
-    "help",
-    "hex",
-    "id",
-    "input",
-    "int",
-    "isinstance",
-    "issubclass",
-    "iter",
-    "len",
-    "list",
-    "locals",
-    "map",
-    "max",
-    "memoryview",
-    "min",
-    "next",
-    "object",
-    "oct",
-    "open",
-    "ord",
-    "pow",
-    "print",
-    "property",
-    "range",
-    "repr",
-    "reversed",
-    "round",
-    "set",
-    "setattr",
-    "slice",
-    "sorted",
-    "staticmethod",
-    "str",
-    "sum",
-    "super",
-    "tuple",
-    "type",
-    "vars",
-    "zip",
-]
+# Code block tags
```

Comment:
This doesn't indeed provide more context than what we can get from the variable names. Would you mind adding some more in-depth description?